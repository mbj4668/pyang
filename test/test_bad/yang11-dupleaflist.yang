module yang11-dupleaflist {

  yang-version 1.1;
  namespace "http://netconfcentral.org/ns/yang11-dupleaflist";
  prefix ydupll;
  revision "2016-11-22";

  leaf-list A {
    type int8;
    default 0;
    default 10;
    default 0; // error; duplicate default value
  }

  leaf-list B {
    config false;
    type int8;
    default 1;
    default 11;
    default 1;
  }

}
