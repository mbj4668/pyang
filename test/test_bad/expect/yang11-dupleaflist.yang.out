yang11-dupleaflist.yang:8: error: DUPLICATE_DEFAULT
